### PR TITLE
docs(contributing): Add GitHub templates for issues, PRs, and contributing

### PR DESCRIPTION
**Pull request by katowulf**
cc @jeffbcross @davideast talk amongst yourselves

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    git push -f
+    ```
+
+That's it! Thank you for your contribution!
+
+#### Deploying Docs
+
+1) Build the docs
+
+`$ npm run docs`
```
Comment (yashmurty) [inline CONTRIBUTING.md]: @katowulf How do I generate the Docs hosted at [AngularFire2 Website](https://angularfire2.com/api/)?
Running `npm run docs` gives me the following error : 
```
npm ERR! Linux 3.13.0-24-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "docs"
npm ERR! node v4.5.0
npm ERR! npm  v3.10.7

npm ERR! missing script: docs
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /var/www/webops/angularfire2/npm-debug.log
```

Comment (davideast) [inline CONTRIBUTING.md]: @yashmurty Those are out of date. We are waiting for TypeDoc to support TypeScript 2.0.

**Comment by davideast** [comment]
@katowulf LGTM!

File: CONTRIBUTING.md (hunk 1 above)
Comment (yashmurty) [inline CONTRIBUTING.md]: Thanks @davideast. Looking forward to contributing on the docs as they seem a little outdated. 
I read somewhere that I need to request to be added as a contributor to the docs. 
Are there steps I need to follow?
Thanks :)

**Pull request merged**